src/Models/HydrostaticFreeSurfaceModels/pcg_implicit_free_surface_solver.jl

```diff
@@ -189,7 +189,7 @@ end
 """
     _implicit_free_surface_precondition!(P_rⁿ⁺¹, grid, r, ∫ᶻ_Axᶠᶜᶜ, ∫ᶻ_Ayᶜᶠᶜ, g, Δt)
 
-Return the simplified inverse preconditioner applied to the residuals in the form
+Return the heuristic inverse preconditioner applied to the residuals in the form
```

Comment:
```suggestion
Return the asymptotic inverse preconditioner applied to the residuals in the form
```

I think a good long description of this preconditioner is that its an asymptotic expansion of the inverse for diagonally-dominant systems.